build_projects/shared-build-targets-utils/BuildCoreClrVersion.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Microsoft.DotNet.Cli.Build
+{
+    public class BuildPackageVersions
+    {
+        public static readonly string CoreCLRVersion = "1.0.2-rc2-24027";
```

Comment:
Yes, I remembered :) 

That is coming in a future PR, I have been moving this thing around a lot, but it should stay here.